src/lib/dremel/tests/Flow/Dremel/Tests/Unit/DremelAssembleTest.php

```diff
@@ -199,75 +199,7 @@ public function test_decode_when_repetitions_definitions_and_values_does_not_hav
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage('repetitions, definitions and values count must be exactly the same');
 
-        \iterator_to_array((new Dremel())->assemble([1, 2], [1], [1, 2]));
-    }
-
-    public function test_decoding_nested_list_with_nulls_and_different_size_of_each_list() : void
```

Comment:
values for those tests were generated with broken shredding logic, they are no longer valid and useful as it's pretty much impossible to understand whats going on here.